src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,23 @@
 Release History
 ===============
 
+2.26.1
+++++++
+
+**ACR**
+
+* Hotfix: `az acr build\connected-registry\pack\run\scope-map`: Fix the compatibility bug caused by SDK upgrade
+
+**AKS**
+
+* Hotfix: `az aks create`: Fix the issue that `assign-kubelet-identity` option can't work
+
+**Storage**
+
+* Hotfix: Fix issue caused by jwt upgrade.
+* Hotfix: `az storage fs directory download`: Fix the issue with `--sas-token` to generate valid sas url
+* Hotfix: `az storage blob copy start`: Fix the issue in copy from different account
```

Comment:
Words like `bug`, `issue` are too general and cannot explicitly reflect the item that is fixed.